omod-1.10/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_10/OrderResource1_10.java

```diff
@@ -39,11 +40,11 @@
 import org.openmrs.module.webservices.rest.web.v1_0.resource.openmrs1_8.OrderResource1_8;
 import org.openmrs.module.webservices.rest.web.v1_0.resource.openmrs1_8.PatientResource1_8;
 
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.Comparator;
-import java.util.Date;
-import java.util.List;
+import io.swagger.models.Model;
+import io.swagger.models.ModelImpl;
```

Comment:
Are these imports necessary for the ticket at hand?

Reply:
Or are they simply formatting changes?

Reply:
@dkayiwa they are formatting changes

Reply:
Can you use a separate pull request for formatting changes?

Reply:
@dkayiwa i have resolved the formatting changes